tests/test_sas7bdat_converter.py

```diff
@@ -1682,6 +1857,91 @@ def test_to_csv_invalid_extension():
     assert "sas7bdat conversion error - Valid extension" in str(execinfo.value)
 
 
+@pytest.mark.parametrize(
+    "fixture_name, expected_name",
+    [
+        ("sas_file_1", "file1.parquet"),
+        ("sas_file_2", "file2.parquet"),
+        ("sas_file_3", "file3.parquet"),
+    ],
+)
+def test_to_parquet_path_sas(tmp_path, fixture_name, expected_name, expected_dir, request):
+    sas_file = Path(request.getfixturevalue(fixture_name))
+    converted_file = tmp_path.joinpath(expected_name)
+    expected_file = expected_dir.joinpath(expected_name)
+    converter.to_parquet(sas_file, converted_file)
+
+    expected = pq.read_table(expected_file)
+    got = pq.read_table(converted_file)
+
+    assert got == expected
+
+
+@pytest.mark.parametrize(
+    "fixture_name, expected_name",
+    [
+        ("xpt_file_1", "file1.parquet"),
+        ("xpt_file_2", "file2.parquet"),
+    ],
+)
+def test_to_parquet_path_xpt(tmp_path, fixture_name, expected_name, request, xpt_expected_dir):
+    xpt_file = Path(request.getfixturevalue(fixture_name))
+    converted_file = tmp_path.joinpath(expected_name)
+    expected_file = xpt_expected_dir.joinpath(expected_name)
+    converter.to_parquet(xpt_file, converted_file)
+
+    expected = pq.read_table(expected_file)
+    got = pq.read_table(converted_file)
+
+    assert got == expected
+
+
+@pytest.mark.parametrize(
+    "fixture_name, expected_name",
+    [
+        ("sas_file_1", "file1.parquet"),
+        ("sas_file_2", "file2.parquet"),
+        ("sas_file_3", "file3.parquet"),
+    ],
+)
+def test_to_parquet_str_sas(tmpdir, fixture_name, expected_name, request, expected_dir):
+    sas_file = request.getfixturevalue(fixture_name)
+    converted_file = str(Path(tmpdir).joinpath(expected_name))
+    expected_file = expected_dir.joinpath(expected_name)
+    converter.to_parquet(sas_file, converted_file)
+
+    expected = pq.read_table(expected_file)
+    got = pq.read_table(converted_file)
+
+    assert got == expected
+
+
+@pytest.mark.parametrize(
+    "fixture_name, expected_name",
+    [
+        ("xpt_file_1", "file1.parquet"),
+        ("xpt_file_2", "file2.parquet"),
+    ],
+)
+def test_to_parquet_str_xpt(tmpdir, fixture_name, expected_name, request, xpt_expected_dir):
+    sas_file = request.getfixturevalue(fixture_name)
+    converted_file = str(Path(tmpdir).joinpath(expected_name))
+    expected_file = xpt_expected_dir.joinpath(expected_name)
+    converter.to_parquet(sas_file, converted_file)
+
+    expected = pq.read_table(expected_file)
+    got = pq.read_table(converted_file)
+
+    assert got == expected
+
```

Comment:
One more testing for missing parquet extras.

```suggestion
def test_to_parquet_missing_openpyxl(tmp_path, sas_file_1):
    with patch("pandas.DataFrame.to_parquet", side_effect=ModuleNotFoundError):
        with pytest.raises(ModuleNotFoundError) as execinfo:
            converter.to_parquet(sas_file_1, tmp_path / "test.parquet")

    assert "parquet extra is required" in str(execinfo.value)
```

Reply:
Thanks for your suggestions, I will work tomorrow to address them.

Reply:
Made requested updates.